VirtoCommerce.Storefront/Controllers/Api/ApiDemoCatalogController.cs

```diff
@@ -0,0 +1,37 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using VirtoCommerce.Storefront.Infrastructure;
+using VirtoCommerce.Storefront.Model;
+using VirtoCommerce.Storefront.Model.Catalog;
+using VirtoCommerce.Storefront.Model.Common;
+using VirtoCommerce.Storefront.Model.Services;
+
+namespace VirtoCommerce.Storefront.Controllers.Api
+{
+    [StorefrontApiRoute("demo")]
+    [ResponseCache(CacheProfileName = "None")]
+    public class ApiDemoCatalogController: StorefrontControllerBase
+    {
+        private readonly IDemoCatalogService _demoCatalogService;
+
+        public ApiDemoCatalogController(IWorkContextAccessor workContextAccessor, IStorefrontUrlBuilder urlBuilder, IDemoCatalogService demoCatalogService) : base(workContextAccessor, urlBuilder)
+        {
+            _demoCatalogService = demoCatalogService;
+        }
+
+        // storefrontapi/catalog/search
+        [HttpGet("catalog/{productId}/configuration")]
+        public async Task<ActionResult<ProductPart[]>> SearchProducts(string productId)
```

Comment:
I think we should call this method something like GetProductConfiguration

Reply:
Fixed